pool/app/message_template/message_template.ml

```diff
@@ -622,3 +622,58 @@ module SignUpVerification = struct
     |> Lwt.return
   ;;
 end
+
+module UserImport = struct
+  let email_address = function
+    | `Admin admin -> Admin.email admin
+    | `Contact contact -> Contact.email_address contact
+  ;;
+
+  let language default_language = function
+    | `Admin _ -> Pool_common.Language.En
+    | `Contact (contact : Contact.t) ->
+      contact.Contact.language |> CCOption.value ~default:default_language
+  ;;
+
+  let email_params confirmation_url user =
+    let fullname = function
+      | `Admin admin -> Admin.full_name admin
+      | `Contact contact -> Contact.fullname contact
+    in
+    [ "name", user |> fullname; "confirmationUrl", confirmation_url ]
```

Comment:
We should consider adding the first-/lastname itself to the params array.

Reply:
Do you think they want to be able to use the first name only? Should we add first name, last name and full name?

Reply:
@mabiede done